api/descriptions/policies/vulnerability_images_get.md

```diff
@@ -1,9 +1,96 @@
-Retrieves the rules that make up your vulnerability policy for container images.
+Retrieves the vulnerability rules for when container images with vulnerabilities are detected, and triggers alerts or blocks deployments.
+
+### cURL Request
+
+The following cURL command retrieves the vulnerability rules for alerts / deployment blocking.
 
 ```bash
 $ curl -k \
   -u <USER> \
   -H 'Content-Type: application/json' \
   -X GET \
-  https://<CONSOLE>:8083/api/v1/policies/vulnerability/images
+  https://<CONSOLE>/api/v1/policies/vulnerability/images?project=<PROJECT>
+```
+
+### Response 
```

Comment:
@KICKdesigns 

I like the example responses as a quick reference, but we've got limited time to maintain them. Next release, we'll be offering customers a Swagger file, which they can load into Postman, point to a live server, and quickly get up-to-date responses. I'm not sure we need these response sections. What's your opinion?

Reply:
@iansk Maintaining does seem like it might be difficult, so perhaps let's just remove them completely for now. Or, we can keep them here for the time being and once the Swagger is available, delete them.

Reply:
@KICKdesigns Let's remove them completely now. That way, I won't have to come back and do it later :)